### PR TITLE
feat(client): use brevo newsletter

### PR DESCRIPTION
**Pull request by DDonochVA**

### CODE REVIEW

File: libs/blog-bff/newsletter/src/lib/api.ts [hunk 1]
```diff
@@ -21,16 +25,57 @@ app.use(
...
+app.post('/subscribe', async (c) => {
   const newSubscriber = await c.req.text();
+  const lang = getWpLang(c);
+  const { BREVO_API_KEY, BREVO_API_URL } = env<{
+    BREVO_API_KEY: string;
+    BREVO_API_URL: string;
+  }>(c);
+
+  let listIds: number[];
+  let templateId: number;
```
Comment (mateuszbasinski) [inline libs/blog-bff/newsletter/src/lib/api.ts:37]: | undefined

File: libs/blog/newsletter/feature-newsletter/src/lib/feature-newsletter/newsletter.component.html [hunk 2]
```diff
@@ -1,62 +1,58 @@
...
-                <label
-                  id="newsletter-label"
-                  class="text-al-muted flex flex-row peer-checked:[&>div>div]:block"
-                  for="newsletter-checkbox"
+      <form role="form" [formGroup]="form">
+        <div class="flex flex-col">
+          <h2 class="text-4xl font-bold">{{ t('title') }}</h2>
+          <p class="py-4">{{ t('description') }}</p>
+          <fieldset>
+            <legend class="hidden">Newsletter sign up form</legend>
```
Comment (mateuszbasinski) [inline libs/blog/newsletter/feature-newsletter/src/lib/feature-newsletter/newsletter.component.html:9]: should this text be static?

**Review by mateuszbasinski — APPROVED** [review]

File: libs/blog-bff/newsletter/src/lib/api.ts (hunk 1 above)
Comment (DDonochVA) [reply libs/blog-bff/newsletter/src/lib/api.ts:37]: will never be undefined, there is 
```ts
default:
      throw new HTTPException(400, { message: 'Invalid lang' });
```
below

File: libs/blog/newsletter/feature-newsletter/src/lib/feature-newsletter/newsletter.component.html (hunk 2 above)
Comment (DDonochVA) [reply libs/blog/newsletter/feature-newsletter/src/lib/feature-newsletter/newsletter.component.html:9]: accessibility

**Pull request merged**